frontend/locales/en/strings.json

```diff
@@ -158,11 +158,33 @@
       "memoryBasedSchedulingEnabled": {
         "label": "Slurm Memory Based Scheduling Enabled",
         "help": "If enabled, users may use <i>--mem</i> to specify the amount of memory per node required by a job. For more information, see the <0>Slurm documentation for ConstrainRAMSpace</0>"
-     },
-     "securityGroups": {
+      },
+      "securityGroups": {
        "label": "Additional Security Groups",
        "help": "Provides additional security groups for the HeadNode."
-     }
+      },
+      "slurmSettings": {
+        "validation": {
+          "databaseCannotBeEmpty": "Database cannot be empty",
+          "databaseDoesNotMatchRegex": "TBD",
```

Comment:
leftover?

Reply:
Yes, going to remove it. We decided to rely on PCluster validation entirely, avoiding to perform regex checks of this kind on PCM side.